code/modules/clothing/head/misc.dm

```diff
@@ -239,6 +241,7 @@
 /obj/item/clothing/head/bandana/orange //themij: Taryn Kifer
 	name = "orange bandana"
 	desc = "An orange piece of cloth, worn on the head."
+	flags_inv = BLOCKHAIR
```

Comment:
Ditto No need for that, /obj/item/clothing/head/bandana is the parent of this object. It already inherits it.

Reply:
If I erase them, the hair doesn't dissapear in the others bandanas